src/Find-SDC-Repositories.js

```diff
@@ -0,0 +1,119 @@
+const graphql = require("graphql-request");
+const { zipWith } = require("lodash");
+const prompt = require("prompt");
+const fs = require("fs");
+const process = require("process");
+const mkdirp = require("mkdirp");
+
+const config = {
+  file: {
+    filePrefix: "SDC-Teams-Github-Repos-",
+    directoryPath: "outputs/SDCTeamGithubRepos/"
+  },
+  api: {
+    endpoint: "https://api.github.com/graphql",
+    options: {
+      headers: {
+        Authorization: null
+      }
+    },
+    query: `
+    query { 
+        organization(login:"ONSDigital") { 
+            teams(first: 100) {
```

Comment:
Maybe extract a constant for the 100 so it's not some magic number e.g. `GITHUB_PAGINATION_LIMIT`?